table-api/src/main/java/io/deephaven/api/updateby/UpdateByOperation.java

```diff
@@ -1733,11 +1743,13 @@ static UpdateByOperation RollingStd(String timestampCol, long revTime, String...
     }
 
     /**
-     * Create a {@link RollingStdSpec rolling standard deviation} for the supplied column name pairs, using time as the
-     * windowing unit. This function accepts {@code nanoseconds} as the reverse and forward window parameters. Negative
-     * values are allowed and can be used to generate completely forward or completely reverse windows. A row containing
-     * a {@code null} in the timestamp column belongs to no window and will not have a value computed or be considered
-     * in the windows of other rows.
+     * Create a {@link RollingStdSpec rolling sample standard deviation} for the supplied column name pairs, using time
+     * as the windowing unit. Sample standard deviation is computed using Bessel's correction:
+     * https://en.wikipedia.org/wiki/Bessel%27s_correction This ensures that the sample variance will be an unbiased
+     * estimator of population variance. This function accepts {@code nanoseconds} as the reverse and forward window
+     * parameters. Negative values are allowed and can be used to generate completely forward or completely reverse
+     * windows. A row containing a {@code null} in the timestamp column belongs to no window and will not have a value
+     * computed or be considered in the windows of other rows.
```

Comment:
In all of these places, the bessel part should be a separate paragraph after the original paragraph.